### PR TITLE
Define license for the project (fixes #225)

### LINKED ISSUE #225: Define license for the project
Discuss and choose which copyleft license we want to have in this project

### PR DESCRIPTION
**Pull request by OtisRed**
As discussed in #225 we chose the MIT license

Please everybody approve/request changes so that we have a clear message on the agreement we stand on

### CODE REVIEW

**Comment by Bosmanfrx** [comment]
Additional thing to consider after merge: Check if Github correctly recognizes license

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 CodeForPoznan
```
Comment (w1stler) [inline LICENSE]: Code for Poznan?

As in: `Fundacja pod nazwą Code for Poznań, zwana dalej „Fundacją”,(...)`

Comment (OtisRed) [reply LICENSE]: @w1stler is that what you've meant?

**Comment by OtisRed** [comment]
@magul there has been limitation for logotypes added in lines 7-8 
@w1stler I believe I addressed your comment properly?

**Pull request merged**